inputFiles/thermoPoromechanicsFractures/plotsolution_thermoPoroelastic.py

```diff
@@ -0,0 +1,102 @@
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import h5py
+import xml.etree.ElementTree as ElementTree
+from mpmath import *
+import math
+
+
+def main():
+    # File paths
+    hdf5FilePathTemperature = "/usr/WS2/cusini1/geosx/geosx_dev/GEOSX/build-quartz-gcc@12-debug/Output/temperature_history.hdf5"
```

Comment:
file path

Reply:
yeah, this is done on purposed. The file should probably just not be in the repo.